README.md

```
@@ -31,51 +26,148 @@ Dive into our [Wiki](https://github.com/Learnathon-By-Geeky-Solutions/binary-bra
-      <td><img src="https://img.shields.io/github/stars/Learnathon-By-Geeky-Solutions/binary-brains.svg?style=for-the-badge&color=4C8BF5" alt="Stargazers" /></td>
-      <td><img src="https://img.shields.io/github/issues/Learnathon-By-Geeky-Solutions/binary-brains.svg?style=for-the-badge&color=FF6F61" alt="Issues" /></td>
-      <td><img src="https://img.shields.io/github/license/Learnathon-By-Geeky-Solutions/binary-brains.svg?style=for-the-badge&color=28A745" alt="License" /></td>
-    </tr>
     <tr>
       <td><img src="https://sonarcloud.io/api/project_badges/measure?project=Learnathon-By-Geeky-Solutions_binary-brains&metric=alert_status&style=for-the-badge&color=4C8BF5" alt="Quality Gate" /></td>
       <td><img src="https://sonarcloud.io/api/project_badges/measure?project=Learnathon-By-Geeky-Solutions_binary-brains&metric=vulnerabilities&style=for-the-badge&color=FF6F61" alt="Vulnerabilities" /></td>
       <td><img src="https://sonarcloud.io/api/project_badges/measure?project=Learnathon-By-Geeky-Solutions_binary-brains&metric=bugs&style=for-the-badge&color=FF6F61" alt="Bugs" /></td>
       <td><img src="https://sonarcloud.io/api/project_badges/measure?project=Learnathon-By-Geeky-Solutions_binary-brains&metric=security_rating&style=for-the-badge&color=28A745" alt="Security" /></td>
       <td><img src="https://sonarcloud.io/api/project_badges/measure?project=Learnathon-By-Geeky-Solutions_binary-brains&metric=code_smells&style=for-the-badge&color=FFA500" alt="Code Smells" /></td>
     </tr>
+   <tr>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: td

(MD033, no-inline-html)